src/ALE/regrid_edge_values.F90

```diff
@@ -240,7 +239,7 @@ subroutine edge_values_explicit_h4( N, h, u, edge_val, h_neglect, answers_2018 )
   real, dimension(4)    :: dz               ! A temporary array of limited layer thicknesses [H]
   real, dimension(4)    :: u_tmp            ! A temporary array of cell average properties [A]
   real, parameter       :: C1_12 = 1.0 / 12.0
-  real                  :: dx, xavg         ! Differences and averages of successive values of x [H]
+  real                  :: dx               ! Differences and averages of successive values of x [H]
```

Comment:
The comment on this line and below at lines 408 and 715 are now inconsistent with the declaration of a single variable.